pages/common/pnmalias.md

```diff
@@ -0,0 +1,20 @@
+# pnmalias
+
+> Apply antialiasing onto a PNM image.
+> More information: <https://netpbm.sourceforge.net/doc/pnmalias.html>.
+
+- Antialias a PNM image, taking black pixels as background and white pixels as foreground:
```

Comment:
This will not fix the error, as the error stems from this line:

https://github.com/tldr-pages/tldr-lint/blob/b5c5a7f56c27e138f229576d363ffbae58a330bd/tldr.l#L158

All words at the beginning of a description ending in `as` are rejected. In other words, @tldr-bot  does not know that `Antialias` is in fact imperative form.

We could do this:

```suggestion
- Perform antialiasing on a PNM image, taking black pixels as background and white pixels as foreground:
```

but I'd be more satisfied if the underlying issue was fixed. However, I have now idea how this could be done.